Makefile.envs

```diff
@@ -147,6 +147,8 @@ export MAIN_MODULE_CFLAGS= $(CFLAGS_BASE) \
 
 export STDLIB_MODULE_CFLAGS= $(SIDE_MODULE_CFLAGS) -I Include/ -I .
 
+export PYODIDE_CMAKE_TOOLCHAIN_FILE=$(PYODIDE_ROOT)/tools/cmake/Toolchain.cmake
```

Comment:
This should not be named `CMAKE_TOOLCHAIN_FILE`, because we don't want to use this toolchain file for library builds like libyaml.